pyart/core/radar.py

```diff
@@ -35,6 +35,7 @@
 from ..config import get_metadata
 from ..lazydict import LazyLoadDict
 from .transforms import antenna_vectors_to_cartesian, cartesian_to_geographic
+from ..util import datetime_utils
```

Comment:
This is going to cause a circular import problem if anything in the util sub-package imports from core, which the xsect module does.  For the time being I would remove this import and the commented out line but keep the TODO line